app/models/autotune/project.rb

```diff
@@ -68,6 +68,14 @@ def publishable?
       draft? || unpublished_updates?
     end
 
+    def live?
+      blueprint_config['preview_type'] == 'live'
+    end
+
+    def set_built
+      update!(:status => 'built')
+    end
```

Comment:
Please remove `set_built`. You're making two SQL updates when you could make just one.